src/test/suite/fixtures/recorded/languages/java/changeCallee4.yml

```diff
@@ -0,0 +1,23 @@
+languageId: java
+command:
+  version: 1
+  spokenForm: change callee
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: functionCallee, includeSiblings: false}
+initialState:
+  documentContents: "public class Test {\r\n    public Test()  {\r\n        super();\r\n    }\r\n}"
```

Comment:
If you set your line endings to `\n` they will be formatted much better in the yml files.

Reply:
Okay will make that change. Do I need to rerecord these tests?

Reply:
I think it would be nice to re-record this one. For the single-line tests it's not necessary, but for multi-line it's much easier to read because then yaml won't wrap it in a string

Reply:
Would it make sense to commit that as a workspace setting so that it's the default for all contributors?

Reply:
Yes!
`"files.eol": "\n"`

Reply:
Unfortunately this didn't work well because the workspace opened when debugging does not use the workspace settings from the actual project.

Reply:
After changing my users level settings it still is not working. I tried restarting the debugger session and deleting then retyping all line breaks to no avail. Not sure what to do about this.

Reply:
I believe that setting is just for new files. If you already saved files with a a different eol those will persist.

Reply:
You can change the eol for an entire document. Look in the footer for `CRLF`.

Reply:
would be nice to re-record with `LF` line endings for readability